src/platform/intel/cavs/lib/pm_memory.c

```diff
@@ -0,0 +1,205 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Jakub Dabek <jakub.dabek@intel.com>
+ */
+
+#include <sof/trace/trace.h>
+#include <sof/lib/memory.h>
+#include <sof/lib/shim.h>
+#include <sof/lib/wait.h>
+#include <sof/lib/pm_memory.h>
+#include <sof/lib/io.h>
+#include <sof/bit.h>
+#include <user/trace.h>
+#include <stdint.h>
+#include <version.h>
```

Comment:
Should be included before standard libraries (due to the rule from local to global), so before #include <stdint.h>.